src/otx/recipe/classification/multi_class_cls/openvino_model.yaml

```diff
@@ -0,0 +1,41 @@
+model:
+  class_path: otx.core.model.entity.classification.OVMulticlassClassificationModel
+  init_args:
+    num_classes: 1000
+    config:
+      class_path: omegaconf.dictconfig.DictConfig
+      init_args:
+        content:
+          model_name: efficientnet-b0-pytorch
+          async_inference: True
+          use_throughput_mode: False
+          model_type: Classification
+          head:
+            num_classes: 1000
```

Comment:
i'm looking forward to refactor this part in the future where we don't pass configs inside configs :)